src/ansys/dpf/composites/_composite_model_impl.py

```diff
@@ -0,0 +1,613 @@
+            mesh=self.get_mesh(),
+        )
+
+    def get_result_times_or_frequencies(self) -> NDArray[np.double]:
+        """Get the times or frequencies in the result file."""
+        return cast(
+            NDArray[np.double], self._core_model.metadata.time_freq_support.time_frequencies.data
+        )
+
+    @_deprecated_composite_definition_label
+    def add_interlaminar_normal_stresses(
+        self,
+        stresses: FieldsContainer,
+        strains: FieldsContainer,
+        composite_definition_label: Optional[str] = None,
+    ) -> None:
+        """Add interlaminar normal stresses to the stresses fields container.
+
+        For a usage example, see
+        :ref:`sphx_glr_examples_gallery_examples_007_interlaminar_normal_stress_example.py`.
+
+        Parameters
+        ----------
+        stresses:
+            Stresses fields container to add interlaminar normal stresses to.
+        strains:
+        composite_definition_label:
+            Label of the composite definition, which is the
+            dictionary key in the :attr:`.ContinuousFiberCompositesFiles.composite`
+            attribute. This parameter is only required for assemblies.
+            See the note about assemblies in the description for the :class:`CompositeModel` class.
+            Interlaminar normal stresses are only added to the layered elements defined
+            in the specified composite definition.
+        """
+        ins_operator = dpf.Operator("composite::interlaminar_normal_stress_operator")
+        ins_operator.inputs.materials_container(self._material_operators.material_provider)
+        ins_operator.inputs.mesh(self.get_mesh())
+        ins_operator.inputs.mesh_properties_container(
+            self._layup_provider.outputs.mesh_properties_container
+        )
+        ins_operator.inputs.section_data_container(
+            self._layup_provider.outputs.section_data_container
+        )
+        ins_operator.inputs.strains_container(strains)
+        ins_operator.inputs.stresses_container(stresses)
+
+        # call run because ins operator has not output
+        ins_operator.run()
+
+    def get_all_layered_element_ids(self) -> Sequence[int]:
+        """Get all layered element IDs."""
+        return cast(
+            List[int],
+            self.get_mesh().property_field("element_layer_indices").scoping.ids,
+        )
+
+    def get_all_layered_element_ids_for_composite_definition_label(
+        self, composite_definition_label: Optional[str] = None
+    ) -> Sequence[int]:
+        """Get all layered element IDs that belong to a composite definition label.
+
+        Parameters
+        ----------
+        composite_definition_label:
+            Deprecated. It's no longer needed
+        """
+        warn(
+            "Call to deprecated function"
+            " get_all_layered_element_ids_for_composite_definition_label."
+            " Please use get_all_layered_element_ids instead.",
```

Comment:
Avoid "please" in technical documentation. Replace with "Use the get_all_layered_element_ids method instead. Maybe consider rewording the message: The ``get_all_layered_element_ids_for_composite_definition_label`` method is deprecated. Use the ``get_all_layered_element_ids`` method instead. 
I'm hoping message strings support literal tagging.